server/tests/auth/test_stateless_auth.py

```
@@ -0,0 +1,157 @@
+        sql = call.args[0] if call.args else None
+        params = call.args[1] if len(call.args) > 1 else None
+        yield sql, params
+
+
+@pytest.fixture(autouse=True)
+def _clear_org_cache(monkeypatch):
+    """Wipe the 5-minute in-process cache so test ordering can't leak state."""
+    monkeypatch.setattr(stateless_auth_module, "_user_org_cache", {})
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)